.github/workflows/setup-and-validate.yml

```diff
@@ -0,0 +1,81 @@
+name: Build and test kotlin
```

Comment:
copied this file from visit-allocation-api. Could use gradle_localstack_postgres_verify.yml but it does not currently allow having custom DB ports which we use.